x-pack/plugins/infra/public/services/telemetry/telemetry_service.test.ts

```diff
@@ -183,4 +183,28 @@ describe('TelemetryService', () => {
       );
     });
   });
+
+  describe('#reportAssetDetailsFlyoutViewed', () => {
+    it('should report asset details viewed with properties', async () => {
+      const setupParams = getSetupParams();
+      service.setup(setupParams);
+      const telemetry = service.start();
+
+      telemetry.reportAssetDetailsFlyoutViewed({
+        componentName: 'infraAssetDetailsFlyout',
```

Comment:
Do we also want to pass the render mode here? Or to have `infraAssetDetailsFlyout` and `infraAssetDetailsPage` to know when a the flyout/details page is rendered